### PR TITLE
Release version increased in pom.xml

### PR DESCRIPTION
**Pull request by shudhansu-shekhar**
### Description of the Change

Release version increased in pom.xml

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: <shudhansu shekhar> <shudhansu.shekhar.ext@ericsson.com>

### CODE REVIEW

File: wiki/markdown/usage/service.md [hunk 1]
```diff
@@ -147,6 +147,9 @@ then in response body we have only one response for that invalid JSON event, it
 
 There is some examples of passing an array of events and what their responses can be
 
+NOTE: Here, introduction of new REMReM property for the size of number of templates in the array passed to generate endpoint. 
+Here, Maximum size of events in array is fetched from REMReM property and checked during generating.
```
Comment (z-sztrom) [inline wiki/markdown/usage/service.md:151]: I think the sentence can be removed, it's implementation detail.

File: wiki/markdown/usage/service.md [hunk 2]
```diff
@@ -147,6 +147,9 @@ then in response body we have only one response for that invalid JSON event, it
 
 There is some examples of passing an array of events and what their responses can be
 
+NOTE: Here, introduction of new REMReM property for the size of number of templates in the array passed to generate endpoint.
```
Comment (z-sztrom) [inline wiki/markdown/usage/service.md:150]: Please, add name of the property.

**Review by z-sztrom — CHANGES_REQUESTED** [review]

File: wiki/markdown/usage/service.md (hunk 1 above)
Comment (shudhansu-shekhar) [reply wiki/markdown/usage/service.md:151]: Done

File: wiki/markdown/usage/service.md (hunk 2 above)
Comment (shudhansu-shekhar) [reply wiki/markdown/usage/service.md:150]: Done

**Review by shudhansu-shekhar — COMMENTED** [review]
Done, please check the latest code

**Pull request merged**